cscs-checks/mch/fieldextra_check.py

```diff
@@ -22,12 +22,10 @@ def __init__(self):
 
 
 @rfm.parameterized_test(['opt'], ['opt_omp'])
-class FieldextraAccuracy(FieldextraCheck):
+class FieldextraAccuracyTest(FieldextraTestBase):
     def __init__(self, variant):
         super().__init__()
-        self.descr = (
-            'Fieldextra validation test (accuracy); MCH'
-        )
+        self.descr = ('Fieldextra validation test (accuracy); MCH')
```

Comment:
No parentheses here.

Reply:
Also remove the `MCH` for all descriptions. You should tag the tests with `mch`.